src/Parser/Druid/Balance/Modules/Features/SunfireUptime.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Enemies from 'Parser/Core/Modules/Enemies';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import { formatPercentage } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class SunfireUptime extends Analyzer {
+  static dependencies = {
+    enemies: Enemies,
+    combatants: Combatants,
+  };
+
+  get suggestionThresholds() {
+    const sunfireUptime = this.enemies.getBuffUptime(SPELLS.SUNFIRE.id) / this.owner.fightDuration;
+    return {
+      actual: sunfireUptime,
+      isLessThan: {
+        minor: 0.95,
+        average: 0.9,
+        major: 0.8,
+      },
+      style: 'percentage',
+      text: 'Your Sunfire uptime can be improved. Try to pay more attention to your Moonfire on the boss.',
```

Comment:
`pay more attention to your Moonfire` should be `pay more attention to your Sunfire`